apigateway/utils.py

```diff
@@ -251,8 +253,20 @@ def _construct_remote_url(self) -> str:
         Returns:
             str: The URL of the remote server.
         """
+        verify_url_regex = re.compile(r"^([12]\d\d\d[A-Za-z&\.]{5}[A-Za-z0-9\.]{9}[A-Z\.]/verify_url\:http[s]?\://.*)")
+
         path = request.full_path.replace(self._deploy_path, "", 1)
         path = path[1:] if path.startswith("/") else path
+        #This block exists because of an incompatibility between urlparse.urljoin and urllib.parse.urljoin
+        #This incompatibility results in http(s):// -> http(s):/ if the proc spec occurs in the middle of the url.
+        try:
+            resolver_check = verify_url_regex.match(path)
+            if resolver_check: 
+                resolver_groups = resolver_check.groups()
```

Comment:
Pretty sure this will only ever return one item in the tuple, since there's only one capturing group in your regex